packages/modules/itwin-error-handling-react/src/components/fallback/FallbackUi.tsx

```diff
@@ -36,7 +38,9 @@ export const FallbackUi = ({
           <Subheading2>{title}</Subheading2>
         </p>
         <p className={styles.message} data-testid="test-fallback-text">
-          <Subheading>{text}</Subheading>
+          <Subheading
+            dangerouslySetInnerHTML={{ __html: innerText }}
+          ></Subheading>
```

Comment:
do you need the separate closing tag?

Reply:
shouldn't now that there's no children. I'll update